README.md

```diff
@@ -30,9 +30,29 @@ docker run -e EMAIL=<EMAIL> -e PASSWORD=<PASSWORD> epicgames
 Replacing the environment variables `EMAIL` and `PASSWORD` for your epicgames store credentials.
 
 ## Optional configuration
-You can also specify two extra environmente variables `TIMEOUT` and `LOGIN_TIMEOUT` if you have a slow internet connection speed and want to make sure it won't affect the result of the script. `LOGLEVEL` can be used to specify the [log level](https://docs.python.org/3.7/library/logging.html#logging-levels) to log.
+You can also specify two extra environmente variables `TIMEOUT` and `LOGIN_TIMEOUT` if you have a slow internet connection speed and want to make sure it won't affect the result of the script. `LOGLEVEL` can be used to specify the [log level](https://docs.python.org/3.7/library/logging.html#logging-levels) to log. `SLEEPTIME` can be used to set the number of seconds to wait between looping through the procedure again. Defaults to -1, which will stop execution after a single iteration.
 Run a docker container with these:
 ```
-docker run -e TIMEOUT=10 -e LOGIN_TIMEOUT=15 -e LOGLEVEL=DEBUG -e EMAIL=<EMAIL> -e PASSWORD=<PASSWORD> epicgames
+docker run -e TIMEOUT=10 -e LOGIN_TIMEOUT=15 -e LOGLEVEL=DEBUG -e SLEEPTIME=43200 -e EMAIL=<EMAIL> -e PASSWORD=<PASSWORD> epicgames
 ```
 These environmente variables have a default value of `TIMEOUT = 5` and `LOGIN_TIMEOUT = 10`.
+
+## Docker Compose
```

Comment:
I am happy with you creating the actual file and not only include an example in the README

Reply:
You mean in addition to the example here? Sure, I can do that.

Reply:
I mean instead of adding the example in here, add the actual file to the repo and then you can mention it... or not :) I guess having a docker-compose may not need any explanation

Reply:
It's nice having it in the readme so that it's visible on Docker Hub. Actually, lots of projects have their docker compose example in the readme. I don't know if it's because some images don't have public git repos associated with them or if it's purely for convenience. Perhaps both.